airbyte-config/config-models/src/main/resources/types/StandardSync.yaml

```diff
@@ -64,11 +86,20 @@ properties:
           - days
           - weeks
           - months
+      # TODO(https://github.com/airbytehq/airbyte/issues/11432): remove. Prefer the basicSchedule property.
       units:
         type: integer
+  # This should be populated when scheduleType is Cron
   # When manual is true, schedule should be null, and will be ignored.
   manual:
     type: boolean
+  # TODO: deprecate/remove `manual`.
+  scheduleType:
+    type: string
+    enum:
+      - Manual
+      - BasicSchedule
+      - Cron
```

Comment:
I'm confused why there are two manuals here. How do we intend to use these?

Reply:
The comments here got a bit mixed up - is the updated version clearer?